readme.md

```diff
@@ -166,34 +166,26 @@ If you want to allow users to [create and manage Jira issues across Mattermost c
 If you face issues installing the plugin, see our [Frequently Asked Questions](#5-frequently-asked-questions-faq) for troubleshooting help, or open an issue in the [Mattermost Forum](http://forum.mattermost.org).
 
 
+## 3. Development
 
-## 3. Jira v2 Roadmap
+To Contribute to the project see https://www.mattermost.org/contribute-to-mattermost
 
-### Timeline
+This plugin contains both a server and web app portion.
+- Use `make dist` to build distributions of the plugin that you can upload to a Mattermost server.
+- Use `make check-style` to check the style.
+- Use `make deploy` to deploy the plugin to your local server.
 
-The ship target dates are included below. These are subject to change:
-  - June 16th: Jira 2.0 released as part of Mattermost Server v5.12
-  - June 25th: Jira 2.1 Release Candidate cut
-       - Deployed to community.mattermost.com for wider testing
-       - Shared with customers for early feedback
-  - August 16th: Jira 2.1 released as part of Mattermost Server v5.14
-  - Oct 16th: Jira 2.2 - bug fixes and some extra troubleshooting functions  
-  - End of October: Jira 2.3 - Improved notification subscription functionality 
+For additional information on developing plugins, refer to [our plugin developer documentation](https://developers.mattermost.com/extend/plugins/).
 
+### Environment Setup
 
-## 4. Development
+This plugin supports both Jira Server (self-hosted) and Jira Cloud instances. There can be slight differences in behavior between the two systems, so it's best to test with both systems individually when introducing new webhook logic, or adding a new Jira API call.
 
-This plugin contains both a server and web app portion.
+To test your changes against a local instance of Jira Server, you need [Docker](https://docs.docker.com/install) installed, then you can use the `docker-compose.yml` file in this repository to create a Jira instance. Simply run `docker-compose up` in the directory of the repository, and a new Jira Server should start up and listen on your localhost's port 8080. It can take a few minutes to start up do to Jira Server's startup processes.
```

Comment:
```suggestion
To test your changes against a local instance of Jira Server, you need [Docker](https://docs.docker.com/install) installed, then you can use the `docker-compose.yml` file in this repository to create a Jira instance. Simply run `docker-compose up` in the directory of the repository, and a new Jira Server should start up and listen on your localhost's port 8080. It can take a few minutes to start up due to Jira Server's startup processes.
```